sql/messaging_system_derived/cfr_users_daily_v1/query.sql

```diff
@@ -0,0 +1,48 @@
+WITH windowed AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    impression_id,
+    client_id,
+    ROW_NUMBER() OVER w1_unframed AS _n,
+    --
+    -- For all dimensions, we use the mode of observed values in the day.
+    udf.mode_last(ARRAY_AGG(release_channel) OVER w1) AS release_channel,
+    udf.mode_last(ARRAY_AGG(locale) OVER w1) AS locale,
+    udf.mode_last(ARRAY_AGG(metadata.geo.country) OVER w1) AS country,
+    udf.mode_last(ARRAY_AGG(version) OVER w1) AS version
+  FROM
+    messaging_system.cfr
+  WHERE
+    -- Reprocess all dates by running this query with --parameter=submission_date:DATE:NULL
+    (@submission_date IS NULL OR @submission_date = DATE(submission_timestamp))
+  WINDOW
+    w1 AS (
+      PARTITION BY
+        sample_id,
+        impression_id,
```

Comment:
Apart from `client_id`, also added `impression_id` here, as release users send impression_id other than client_id for CFR. This also applies to the Whats-new-panel table.

@jklukas does it make sense?

Reply:
Yes, this looks great.